packages/govern-subgraph/src/ERC3000Registry.ts

```diff
@@ -14,16 +14,16 @@ import {
 import { loadOrCreateGovern } from './Govern'
 
 export function handleRegistered(event: RegisteredEvent): void {
-  const registry = loadOrCreateRegistry(event.address)
+  let registry = loadOrCreateRegistry(event.address)
```

Comment:
Why we choose to use `let` instead of `const`?

Reply:
PR #102 explains why :-)